src/engine/server_iv.c

```diff
@@ -607,7 +607,7 @@ ivc_on_put(crt_iv_namespace_t ivns, d_sg_list_t *iv_value, void *priv)
 	D_ASSERT(entry != NULL);
 
 	/* Let's deal with iv_value first */
-	d_sgl_fini(iv_value, false);
+	d_sgl_fini(iv_value, true);
```

Comment:
@jolivier23 What do you think about this?

Reply:
Buffers are dynamically allocated, so they should be released.

Reply:
I thought he'd said previously they were supposed to be released at a higher level.